pelicanconf.py

```diff
@@ -25,6 +29,15 @@
 AUTHOR_FEED_ATOM = None
 AUTHOR_FEED_RSS = None
 
+# Method to call elements from json files
+def getContentElements(path):
```

Comment:
O ideal para o nome da função (segundo a PEP8) é usar snake_case em vez de camelCase. Sugiro também mudarmos o nome da função para ficar mais explícito:

```suggestion
def get_authors_data(path):
```